output.go

```diff
@@ -42,6 +42,31 @@ func FormatYaml(writer io.Writer, value interface{}) error {
 	return nil
 }
 
+// FormatYaml writes out value as yaml to the writer, unless value is nil.
+func FormatErrYaml(writer io.Writer, value interface{}) error {
+	if value == nil {
+		return nil
+	}
+	msg := value.(error).Error()
+	result, err := goyaml.Marshal(ErrorResponse{ErrorMsg: msg})
+	if err != nil {
+		return err
+	}
+	for i := len(result) - 1; i > 0; i-- {
+		if result[i] != '\n' {
+			break
+		}
+		result = result[:i]
+	}
```

Comment:
I don't think you need to trim do you? I'm unsure why you'd end up with lots of lines. There is no way this will work if you add more members to `ErrorResponse`.